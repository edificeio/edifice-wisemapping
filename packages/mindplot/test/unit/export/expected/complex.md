# PPM Plan

- Business Development 
- Backlog Management ( [link](https://docs.google.com/a/freeform.ca/drawings/d/1mrtkVAN3_XefJJCgfxw4Va6xk9TVDBKXDt_uzyIF4Us/edit) )
- Freeform IT
- Client Project Management
- Governance & Executive
- Finance
- Administration
- Human Resources[^1] 
- Freeform Hosting
- Community Outreach
- R&D
	- Goals
	- Formulize
- Probono



[^1]: HR Vision: Freeform Solutions is successful at its mission, sustainable as an organization AND is a great place to work.HR Mission: To provide a positive HR service experience for applicants and employees, and  collaborate with departments to recruit, develop, support, and retain diverse and talented employees who are the key to Freeformâ€™s  reputation and success.
