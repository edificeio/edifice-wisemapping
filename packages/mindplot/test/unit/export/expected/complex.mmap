<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="1W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Finance"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2064nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Governance &amp; Executive"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Administration"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Backlog Management"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/><ap:Hyperlink Url="https://docs.google.com/a/freeform.ca/drawings/d/1mrtkVAN3_XefJJCgfxw4Va6xk9TVDBKXDt_uzyIF4Us/edit"/></ap:Topic><ap:Topic OId="2044nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Client Project Management"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2684nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2694nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Probono"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1054nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Freeform IT"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2474nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Community Outreach"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1544nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Human Resources"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1654nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Freeform Hosting"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Business Development "><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2614nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2634nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Goals"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2644nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Formulize"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="R&amp;D"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="PPM Plan"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/></ap:Topic></ap:OneTopic><ap:Relationships/></ap:Map>