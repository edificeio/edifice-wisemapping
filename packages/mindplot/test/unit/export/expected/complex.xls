<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Excel.Sheet"?><Workbook xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet" xmlns:duss="urn:schemas-microsoft-com:office:dummyspreadsheet" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="urn:schemas-microsoft-com:office:spreadsheet">
<Styles>
<Style ss:ID="s16" ss:Name="attribute_cell">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
</Style>
<Style ss:ID="s17" ss:Name="attribute_header">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
<Font ss:Bold="1"/>
</Style>
</Styles>
<Worksheet ss:Name="FreeMind Sheet">
<Table>
<Row>
<Cell ss:Index="1">
<Data ss:Type="String">PPM Plan</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Finance</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Governance &amp; Executive</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Administration</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Backlog Management</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Client Project Management</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Probono</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String"/>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Freeform IT</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Community Outreach</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Human Resources</Data>
<Comment>
<ss:Data xmlns="http://www.w3.org/TR/REC-html40">
<p/>
</ss:Data>
</Comment>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Freeform Hosting</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Business Development </Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">R&amp;D</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Goals</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Formulize</Data>
</Cell>
</Row>
</Table>
</Worksheet>
</Workbook>
