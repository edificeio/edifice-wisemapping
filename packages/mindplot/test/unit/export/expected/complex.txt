1 PPM Plan
	1.1 Business Development 
	1.2 Backlog Management
 	  [Link: https://docs.google.com/a/freeform.ca/drawings/d/1mrtkVAN3_XefJJCgfxw4Va6xk9TVDBKXDt_uzyIF4Us/edit]
	1.3 Freeform IT
	1.4 Client Project Management
	1.5 Governance & Executive
	1.6 Finance
	1.7 Administration
	1.8 Human Resources
	  [Note: HR Vision: Freeform Solutions is successful at its mission, sustainable as an organization AND is a great place to work.
HR Mission: To provide a positive HR service experience for applicants and employees, and  collaborate with departments to recruit, develop, support, and retain diverse and talented employees who are the key to Freeform‚Äôs  reputation and success.]
	1.9 Freeform Hosting
	1.10 Community Outreach
	1.11 R&D
		1.11.1 Goals
		1.11.2 Formulize
	1.12 Probono
		1.12.1 
2 Strategy 2: Talent Development
  [Note: Strategy #2: Support the talent development of our employees through professional development and learning and through improved performance management.]
	2.1 Strategic Priority 2a: Personal Plans
	  [Note: Each employee will have a personal Professional Development Plan.  ]
	2.2 Strategic Priority 2b: External learning matches organ. goals
	  [Note: Each department of Freeform will identify areas that need development to meet overall FS goals.  Eg.  Project Manager may identify needed improvement in a development tool.  Or... Bus. Dev. may identify a new need in NFP that FS could fill within mandate, if training were provided.  Professional Dev. priority will be given to proposals for development with clear ROIs.]
	2.3 Strategic Priority 2c: Learning Environment
	  [Note: Learning and innovation are an essential part of providing the best solutions to NFPs.  Cost effective internal learning and time to explore innovation will be encouraged, provided they conform with organization goal and clear ROI is demonstrated.]
	2.4 üåà So That...
	  [Note: (So that... our employees have improved skills and knowledge, So that... they are highly competent and can work well in agile teams and feel fulfilled and self actualized... So that we can so the best work possible, for the least cost, in the shortest time for other NFPs, So that... NFPs can help those who need it.)]
3 Strategy 4: Inclusive, Positive Environment
  [Note: Strategy #4: Foster a diverse, inclusive community with a positive work environment.]
	3.1 Strategic Priority 4a:Feedback
	  [Note: Conduct regular organizational feedback assessments and collaborate to improve the work climate]
	3.2 Strategic Priority 4b: Anti Harassment
	  [Note: Educate employees on the prevention of harassment and discrimination and productive ways to resolve conflict]
	3.3 Strategic Priority 4c: Diversity
	  [Note: Insure we promote our commitment to diversity and non-discrimination through our actions and in our outreach and employee recruitment efforts]
	3.4 
	3.5 üåà So That...
	  [Note: (So that... we can reflect the diverse populations we serve AND ensure everyone feels safe, respected  and  included, So that... we better serve our diverse client organizations AND we are a great place to work )]
4 Strategy 1: Recruit & Retain
  [Note: Recruit and retain top talent commensurate with identified organizational capacity requirements ]
	4.1 So that...
	  [Note: (So that... we find and keep good people, So that... they are highly competent and can work well in agile teams... So that we can so the best work possible, for the least cost, in the shortest time for other NFPs, So that... NFPs can help those who need it.)]
	4.2 Strategic Priority 1a: Recruitment
	  [Note: 1. Identify and use proactive and effective recruitment strategies, ]
		4.2.1 Modify App Form
		  [Note: Recently, I saw a few job posts sent through different community
groups and they seem to be taking our idea of screening candidates
to a next level. Not only they ask candidates to provide resume and
cover letter + some project related information but also request
written answers to questions like "Why are you interested in this
position" etc. That allows to screen out people who just submit
multiple resumes to multiple organizations without really applying
for that particular job and it would make our interview process more
straightforward, knowing answers to some questions.

For example, we may want to always include in the screening
questions:
- Why do you want to work for Freeform Solutions?
- Why are you interested in this position?
- What's your experience working with NFP?
- What's your experience working with Open Source software?
etc.

I also saw that people ask for references up front, in that
submissions form. We could include the HR requirement that Heather
brought recently for "permissions to ask your references about you"
in the online form so that we don't have to follow up with this
later.

Attached below a sample of such screening questions]
		4.2.2 Strategy integrated with hiring plan
		  [Note: Hiring plan should be comprehensive... not Agile or Iterative, in the sense that staff capacity and skill needs should be met within at least a six month plan.  If three Drupal developers are needed, the hiring should be done concurrently to minimize HR costs and time.]
	4.3 Strategic Priority 1b: Hiring
	  [Note: 2. Continue to practice our unique Freeform hiring process that balances fit with the Freeform culture and best talent ]
	4.4 Strategic Priority 1c: Onboarding
	  [Note:  ]
		4.4.1 3 Month Onboarding Process
		4.4.2 Tools & Guidelines
		4.4.3 Mentoring
	4.5 Strategic Priority 1d: Incentives
	  [Note: 5. Explore incentives - monetary, benefits, fulfilment - needed to encourage top talent to work for and remain working for an NFP]
		4.5.1 Raises
		4.5.2 Benefits
		4.5.3 Rewards Message
		  [Note: Create a total rewards message to encourage prospective and current employees to understand the full value of working for Freeform]
	4.6 Strategic Priority 1e: Offboarding
	  [Note: Assess and address reasons why talented people leave Freeform]
5 Business Development Plan
	5.1 Goals
		5.1.1 Increase new clients
			5.1.1.1 Academic Research
			5.1.1.2 
		5.1.2 Support New Products
			5.1.2.1 Formulize
			5.1.2.2 
			5.1.2.3 
		5.1.3 Support CiviCRM
		5.1.4 Identify Opportunites
			5.1.4.1 
			5.1.4.2 
			5.1.4.3 
			5.1.4.4 
6 Hosting NG Plan
7 Freeform IT Plan
	7.1 Fragile
	7.2 Tools
	7.3 
8 Project Teams
	8.1 Projects 1-3
	8.2 Projects 4-6
	8.3 Projects 7 & 8
	8.4 General Work
	8.5 Learning Needs Plan
9 Restructure
	9.1 Client Centric Process
	9.2 Freeform Project Process
	9.3 Supportive Systems Plan
10 Board and C Planning
	10.1 Mission Statements
	  [Note: In the absence of one clearly defined mission statement, we have reviewed various expressed mission statement as following

    Objects of Incorporation in Letters Patent
        The objects of the Corporation are:To provide solutions that facilitate the effective use of information technology in not-for-profit, non-governmental, and charitable organizations throughout Canada, to support and improve their mission delivery.
    2006 Strategic Business Plan - Brand Positioning
        We are a nonprofit dedicated to helping other nonprofits understand and employ technology appropriately and effectively to support their mission. Our mission is to provide flexible consulting, website development, and Internet hosting solutions that give you peace of mind and help you stay focused on your mission.
    Freeform One Page (Freeform Wiki)
        About Freeform Solutions: Freeform Solutions is a not-for-profit organization. Our mission is help (sic) other not-for-profits organizations to build their capacity and increase their effectiveness.
    Result of Google search for ‚ÄúFreeform Solutions mission‚Äù
        Freeform Solutions is a not-for-profit organization. Our mission is to help other not-for-profits use technology to build their capacity and increase their effectiveness. (www.freeformsolutions.ca/en/files/AboutFreeformSolutions.pdf
        Freeform Solutions (www.freeformsolutions.ca) is a non profit organization. Our mission is to help other non-profit organizations to realize their missions through the appropriate deployment of information and knowledge management systems.www.freeformsolutions.ca/en/sites/default/.../virtual.volunteering.pdf
        We are a not-for-profit organization (NFP) that helps other NFPs use IT to achieve their organizational goals and better serve their communities. Freeform Solutions is a nonprofit organization dedicated to helping other nonprofit organizations understand and employ technology appropriately and effectively to support their missions. Our mission is to provide flexible consulting, website development, and hosting solutions  http://socialinnovation.ca/community/organizations/freeform-solutions
        Freeform Solutions is a not-for-profit organization, with a mission to help other not-for-profits use technology more effectively to meet their own missions. http://timreview.ca/article/387
        Freeform Solutions ‚Äì their mission: ‚Äúwe help not-for-profit organisations use technology to build their capacity and increase their effectiveness. http://www.warnerlaw.ca/links/community-organisations-and-local-businesses/
        Freeform Solutions is a not-for-profit organization with a mission to help other not-for-profits use technology to meet their goals. http://xoops.org/modules/news/article.php?storyid=3860
        At Freeform Solutions, we have a mission to help not-for-profit and public sector organizations use technology more effectively. http://osbrca.blogspot.ca/2010/07/development-commons-approach.html
    Freeform Solutions Facebook Page
        Mission: Freeform Solutions is a not-for-profit organization (NFP) that helps other NFPs use IT to achieve their organizational goals and better serve their communities.
    The current Freeform Solutions website
        We started Freeform to help NFPs use IT to achieve their organizational goals and better serve their communities - to support and improve their mission delivery. Our mission is to strengthen the capacity of NFPs and the voluntary sector, and to help build a civil society.]
	10.2 Values
	10.3 Bylaw Review
	10.4 Policies
	10.5 Business Plan
11 Strategy 3: Safety and Wellness
  [Note: Strategy # 3: Promote the achievement of safety and wellness in our virtual employee community.]
	11.1 Strategic Priority 3a: H&S Policies & Practices
	  [Note: Continuing improvement in Health and Safety policies and practices & compliance with OHSC legislation]
		11.1.1 
	11.2 Strategic Priority 3b: Health Promotion
	  [Note: Promoting safety, work-life balance, self-care, ergonomics and other factors for wellness and productivity in a virtual workplace environment]
		11.2.1 Health and Wellness Committee
		  [Note: The Freeform H&S rep will lead a Health and Wellness Committee to  responsible for recognizing health and safety concerns and identifying solutions.]
		11.2.2 Work-life Balance Initiative
 		  [Link: http://hrcouncil.ca/hr-toolkit/workplaces-health-safety.cfm]
	11.3 üåà So that...
	  [Note: (So that... our employees remain well and safe, So that... they are highly competent and can work well in agile teams and feel fulfilled and self actualized... So that we can so the best work possible, for the least cost, in the shortest time for other NFPs, So that... NFPs can help those who need it.)]
12 Benefits
  [Note: As Freeform Employees we will have benefits reviewed in light of our priorities and cost to Freeform]
	12.1 üë®‚Äçüëß‚Äçüë¶ As Freeform Staff
	12.2 Responsibility: HZ, JC
	12.3 Release 3
	12.4 ‚úÖ Have Heather create list benefits against Best Practice & cost
	12.5 Have Jason  review list
	12.6 Have JC & HZ consult with staff
	12.7 Have best benefits we can afford
	12.8 üåà So that...
	  [Note:  so that our efforts to excel are rewarded.]
13 Community Outreach Plan
	13.1 Goals
	13.2 CSI
	13.3 Drupal Community
	13.4 CiviCRM
	13.5 Other
14 Backlog Plan
   [Link: https://docs.google.com/a/freeform.ca/drawings/d/1mrtkVAN3_XefJJCgfxw4Va6xk9TVDBKXDt_uzyIF4Us/edit]
	14.1 üòé Go To Backlog Plan 
 	  [Link: https://docs.google.com/a/freeform.ca/drawings/d/1mrtkVAN3_XefJJCgfxw4Va6xk9TVDBKXDt_uzyIF4Us/edit]
15 Strategy Prospecting
	15.1 
	15.2 
	15.3 
16 Stategies:  Forecasting
	16.1 
	16.2 
	16.3 
17 Strategies Marketing
18 
19 Exit Interviews
	19.1 üë®‚Äçüëß‚Äçüë¶ As Freeform
	19.2 Responsiblity: HZ, KS
	19.3 ‚åõ Release
	19.4 Have Heather write procedures for exit interview process
	19.5 üåà So that
	  [Note: We learn from our mistakes and missed opportunities in future with the goal of keeping the best talent.]
20 3 Month Onboarding Process
21 Human Resources Plan
	21.1 Related Org Objectives
		21.1.1 1
		  [Note: Attract, build and retain a motivated, agile, knowledgeable team of Top Talent that loves to ‚Äúcome‚Äù to work]
		21.1.2 2
		  [Note: Maintain level of human resource capacity and skill to meet planned growth and client contractual commitments]
		21.1.3 3
		  [Note: Conform to all legislated requirements]
		21.1.4 4
		  [Note: Minimize and mitigate risk to the organization]
	21.2 Related Documents
	  [Note: MIssion, Values, Principles, Org Business Plan, Human Resources Policy Manual]
	21.3 Goals
		21.3.1 üèà Goal:Staff=Optimal Bus. Growth
		  [Note: Human resource capacity will remain at a level to  meet planned growth growth objectives and client contractual commitments]
			21.3.1.1 üåà So that...
			21.3.1.2 Related Strategic Priorities: 
			21.3.1.3 üìä KPI: HR Level equals Planned Growth
			21.3.1.4 Methodology
			  [Note: Schedule of required HR capacity vs. actual HR capacity.  Variance + or - 1]
				21.3.1.4.1 Target
				  [Note: = + or - 1]
		21.3.2 üèà Goal: Increase Job Satisfaction
			21.3.2.1 üåà So That
			  [Note:     Establish better relationships.
    Identify with the new employer.
    Build a great attitude with the company.]
			21.3.2.2 Related Strategic Priorities
				21.3.2.2.1 
			21.3.2.3 üìä KPI: Employee Satisfaction
				21.3.2.3.1 
			21.3.2.4 Methodology
			  [Note: Percentage of improvement in employee reported job satisfaction based on survey vs previous year.  Base level to be established in first year. ]
				21.3.2.4.1 Target
				  [Note: Base level 1st  year]
		21.3.3 üèà Goal: Improve Performance
		  [Note: To increase knowledge, skills and experience of the Freeform staff relevant to organizational priorities.]
			21.3.3.1 üåà So That
			  [Note:     Clarify expectations.
    Understand values and priorities.
    Decrease the learning curve.]
			21.3.3.2 Related Strategic Priorities
			  [Note: 1]
			21.3.3.3 üìä KPI: Employee Performance
			21.3.3.4 Methodology
				21.3.3.4.1 Target
		21.3.4 üèà Goal: Reduce Turnover
		  [Note: To reduce turnover of Top Talent.]
			21.3.4.1 üåà So That
			  [Note:     Provide support through feedback.
    Help the employee feel valued.
    Again, decrease the learning curve.]
			21.3.4.2 Related Strategic Priorities
			21.3.4.3 üìä KPI: Retention Rate
			21.3.4.4 Methodology
				21.3.4.4.1 Target
		21.3.5 üèà Risk & Compliance
		  [Note: To eliminate or minimize risk and to comply with all legislated requirements.  ]
